autofill/autofill-impl/src/main/res/layout/content_autofill_generate_password_dialog.xml

```diff
@@ -43,18 +44,33 @@
 
         <com.duckduckgo.mobile.android.ui.view.text.DaxTextView
             android:id="@+id/generatedPassword"
-            android:layout_width="0dp"
+            android:layout_width="wrap_content"
+            app:layout_constrainedWidth="true"
             app:typography="body1_mono"
             android:gravity="center_horizontal"
-            android:textIsSelectable="true"
-            android:longClickable="true"
+            android:layout_marginStart="@dimen/keyline_1"
+            android:layout_marginEnd="@dimen/keyline_1"
             android:layout_marginTop="28dp"
+            tools:text="abc123ABC456@^*789abc123ABC456@^*789abc123ABC456@^*789"
             android:layout_marginBottom="28dp"
             app:layout_constraintEnd_toEndOf="@id/useSecurePasswordButton"
             app:layout_constraintStart_toStartOf="@id/useSecurePasswordButton"
             app:layout_constraintTop_toBottomOf="@id/dialogTitle"
             android:layout_height="wrap_content" />
 
+        <ImageButton
```

Comment:
I believe we should be using `IconButton` from the design system. cc/ @malmstein